src/knex-tables.d.ts

```diff
@@ -148,10 +150,75 @@ declare module "knex/types/tables" {
     id: number;
     onerep_profile_id: number;
     onerep_scan_id: number;
-    onerep_scan_results: ScanResult;
+    onerep_scan_reason: Scan["reason"];
     created_at: Date;
     updated_at: Date;
   }
+  type OnerepScanOptionalColumns = Extract<
+    keyof OnerepScanRow,
+    "onerep_profile_id"
+  >;
+  type OnerepScanAutoInsertedColumns = Extract<
+    keyof OnerepScanRow,
+    "id" | "created_at" | "updated_at"
+  >;
+
+  interface OnerepScanResultRow {
+    id: number;
+    onerep_scan_result_id: number;
+    onerep_scan_id: OnerepScanRow["onerep_scan_id"];
+    link: string;
+    age?: number;
+    data_broker: string;
+    data_broker_id: number;
+    emails: string[];
+    phones: string[];
+    addresses: Array<{
+      city: string;
+      state: StateAbbr;
+      street?: string;
+      zip?: string;
+    }>;
+    relatives: string[];
+    first_name: string;
+    middle_name?: string;
+    last_name: string;
+    status: RemovalStatus;
+    created_at: Date;
+    updated_at: Date;
```

Comment:
same as above re append-only

Reply:
That's a good point. Removed the column and blocked update queries in the type in ca99aa2c7.

Reply:
Oh actually, I removed this again: 1f4681b34

The entire point of this change was to prepare https://github.com/mozilla/blurts-server/pull/3385, where I _do_ update scan result rows when they are manually resolved :sweat_smile:

Reply:
Ok that makes sense